pymc3/step_methods/nuts.py

```diff
@@ -136,6 +137,14 @@ def astep(self, q0):
 
         return q
 
+    @staticmethod
+    def competence(var):
+        if var.dtype in continuous_types:
+            if sum(var.shape) > 5:
```

Comment:
Simplest solution is to simply prefer NUTS for any continuous variable, and let users pick Metropolis manually if they want it.